src/ConfigManager.cpp

```diff
@@ -235,7 +248,7 @@ void ConfigManager::clearWifiSettings(bool reboot) {
   memset(password, 0, PASSWORD_LENGTH);
 
   DebugPrintln(F("Clearing WiFi connection."));
-  storeWifiSettings(ssid, password, true);
+  storeWifiSettings(ssid, password, false);
```

Comment:
I didnt see the reason that this was begin flipped. Maybe it was a remnant of some code changes ? Now Im thinking this is more of the flag that the memory has been initialized?